src/karma-webpack.js

```diff
@@ -270,7 +233,9 @@ Plugin.prototype.make = function(compilation, callback) {
       let entry = file;
 
       if (this.wrapMocha) {
-        entry = `${require.resolve('./mocha-env-loader')}?name=${compilation.name}!${entry}`;
+        entry = `${require.resolve('./mocha-env-loader')}?name=${
```

Comment:
Seems prettier wanted to change this line on this commit for some reason..

Reply:
👍